podcast/2.13.md

```diff
@@ -0,0 +1,913 @@
+
+And so that's where we started, but we always had this vision. The original vision for Sourcegraph was just to build this single portal where all the code in the world and the code that you're interested in or might be interested in specifically is accessible from a single search box. And so, a big effort for us right now is kind of warming this community back up because it was just not a focus for our company for a number of years. But we think it is central to our mission and what we want to do. And I would love to hear your advice for both us and other developer tool creators and founders here about how to build and foster that strong technical community.
+
+**swyx:**
+
+Yeah. I don't have zero experience, but I'm also not the world expert on this thing. So I should probably list out the stuff that I've done, right? So, I helped to grow the React reddit from 30 to 220,000. And then, I left React and started Svelte Society from zero. I helped from the first meeting. And we're about to hit 10,000 people.
+
+And then, I was part of Netlify as we established the Jamstack community. And now, we're trying to grow that at Temporal as well. I think that we have a similar issue with Temporal as you do. We're adopted by people with microservice orchestration problems, which is generally large companies. So, it's a different topic for a front end community where you can be productive with just a single person on a browser. The advice that I typically give to people, because a lot of people definitely reached out to me after that blog post is: hire from your community rather than trying to install someone with a proven track record in a different community, because I don't think it transfers that well. Because they might just fundamentally not get the community. And you want someone who already embodies the kind of person that you want more of, right?
+
+So I always say, "Pick the community member who you would most like to clone." Because essentially, if you make that their whole job, then they will start cloning themselves in whatever way. It could be from product decisions, marketing messaging, to just the networks that they're already in. They serve as a beacon or lighthouse for people like them to come on in. And so, there's good and bad. You will necessarily exclude some people just as a result of unintentional consequences. So, you do have to pick somewhat carefully. But what I'm basically saying not to do is say, "Okay, this person has 10 years managing community at, name your other startup. Let's just hire them. They've done it before they can do it again here."
+
+I don't think that works because they may not get it.
+
+**Beyang Liu:**
+
+Because so much of the qualification for the role is just understanding your specific tool and how it impacts the user's lives, right?
```

Comment:
```suggestion
Because so much of the qualification for the role is just understanding your specific tool and how it impacts the users' lives, right?
```